src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1219,13 +1252,16 @@ public void UpdateBubbleContent()
             foreach (var info in NodeModel.Infos)
             {
                 var infoStyle = info.State == ElementState.Error ? InfoBubbleViewModel.Style.Error : InfoBubbleViewModel.Style.Warning;
+                infoStyle = info.State == ElementState.Info ? InfoBubbleViewModel.Style.Info : infoStyle;
```

Comment:
I don't think the info state should override the error state ⚠️

Reply:
Modified this to follow the 1.Error 2. Warning 3. Info hierarchy.